jobs/JREGIONAL_PROCESS_BUFR

```diff
@@ -72,7 +72,7 @@ mkdir_vrfy -p ${WORKDIR}
 #-----------------------------------------------------------------------
 #
 $SCRIPTSDIR/exregional_process_bufr.sh \
-  CYCLE_DIR="${CYCLE_DIR}" WORKDIR="${WORKDIR}" || \
+  CYCLE_DIR="${CYCLE_DIR}" WORKDIR="${workdir}" || \
```

Comment:
If to follow the rule global uppercase local lowercase for variables, should the "CYCLE_DIR", "WORK_DIR" before "=" be in lowercase?  

I checked  jobs/JREGIONAL_RUN_FCST and it looks like so:
$SCRIPTSDIR/exregional_run_fcst.sh \
       cdate="${CDATE}" \
       cycle_dir="${CYCLE_DIR}/fcst_fv3lam" \
...

However, I don't quite get it why we need to pass $CDATE into exregional_run_fcst.sh through $cdate since $CDATE is a global variable and can be accessed by the script?  @christinaholtNOAA could you give more insights to this? Thanks!

Reply:
I kept WOKRDIR to show it is a variable passed through to run script.